contents/tutorials/aa-testing.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to do A/A testing
+date: 2023-07-06
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-4.png
+tags: ['experimentation', 'feature flags']
+---
+
+An A/A test is the same as an [A/B test](/docs/experiments) except both groups getting the same code or component. Because both groups get functionally identical variants, you want to see no statistical difference between the variants by the end of the experiment. If there is a significant difference, that is cause to investigate.
+
+Teams run A/A tests to ensure their A/B test service, functionality, and implementation process results in accurate and relevant results. A/A tests are a way to discover broken parts of this process, and this tutorial shows you how to run one in PostHog.
+
+> Unlike other A/B testing tools, A/A tests aren’t needed to calculate baseline conversion and sample metrics in PostHog. You can calculate those using your product data, and they are automatically when you create an experiment.
```

Comment:
Running an A/A test is required in other tools?? I found that very surprising

Reply:
Yeah a couple of the YC examples needed to use A/A tests to figure out how long to run tests for.